collection-scripts/gather

```diff
@@ -15,11 +15,13 @@ OPERATOR_NS=${OPERATOR_NAMESPACE:-redhat-ods-operator}
 NOTEBOOKS_NS=${NOTEBOOK_NAMESPACE:-rhods-notebooks}
 MONITORING_NS=${MONITORING_NAMESPACE:-redhat-ods-monitoring}
 APPLICATIONS_NS=${APPLICATIONS_NAMESPACE:-redhat-ods-applications}
+MODELREG_NS=${MODELS_REGISTRY_NAMESPACE:-rhoai-model-registries}
 
 oc adm inspect $log_collection_args "namespace/$OPERATOR_NS" --dest-dir="$DST_DIR" || echo "Error getting logs from ${OPERATOR_NS}"
-oc adm inspect $log_collection_args "namespace/$NOTEBOOKS_NS" --dest-dir="$DST_DIR" || echo "Error getting logs from ${NOTEBOOKS_NS}"
 oc adm inspect $log_collection_args "namespace/$MONITORING_NS" --dest-dir="$DST_DIR"  || echo "Error getting logs from ${MONITORING_NS}"
 oc adm inspect $log_collection_args "namespace/$APPLICATIONS_NS" --dest-dir="$DST_DIR"  || echo "Error getting logs from ${APPLICATIONS_NS}"
+oc adm inspect $log_collection_args "namespace/$NOTEBOOKS_NS" --dest-dir="$DST_DIR" || echo "Error getting logs from ${NOTEBOOKS_NS}"
```

Comment:
Is there a semantic reason for the move of the notebook line move, or just making things nicer?

Reply:
order them in priority, since notebook namespace might not be in use if user opt-in their own namespaces rather than default one.

Reply:
Makes sense ... but I wonder if we should reorder things elsewhere (in the README, for example) as well, in a separate commit. /nitpick :-)

Reply:
yeah, i agree. 
could not help myself..... :rofl: